fontbe/src/orchestration.rs

```diff
@@ -248,41 +248,39 @@ impl TupleBuilder {
     }
 }
 
-#[repr(u8)]
-#[derive(Copy, Clone, Debug, PartialEq)]
-pub enum LocaFormat {
-    Short = 0,
-    Long = 1,
+// work around orphan rules.
+//
+// FIXME: Clarify if there's a good reason not to treat glyf/loca as a single
+// entity, for the purpose of persistence? like a struct that contains both
+// tables, and from which the format can be retrieved
```

Comment:
Given that we compute them together I think this might make more sense and might be a better change. Want to try that?

Reply:
Also I really dislike the format wrapper; hopefully use of a single entity would let us avoid that

Reply:
yea, it felt very bad writing this code. That said I think it makes sense to merge as is and open an issue, and that work can be a separate patch.